app/views/application/_pg_header.html.erb

```diff
@@ -0,0 +1,15 @@
+<% config = false unless config %>
```

Comment:
I thought it will be necessary to make make clear that the value contains a true/false value. But it is true, it is very redundant, it will be changed as well.